images/build/go-runner/variants.yaml

```diff
@@ -15,11 +15,3 @@ variants:
     REVISION: '0'
     GO_VERSION: '1.21.7'
     DISTROLESS_IMAGE: 'static-debian12'
-  go1.20-bullseye:
```

Comment:
should this have changed to go1.21-bullseye?

Reply:
go runnuer on go1.21 is using bookworm, the release-1.29 is using go-runner `v2.3.1-go1.21.7-bookworm.0`

do you want to build go-runner for releases 1.28/27/26 with bullseye? or keep the same we are using for release-1.29?

Reply:
I... don't know enough to know for sure, but I like changing one variable at a time if possible, so I sort of expected to keep using bullseye for 1.26 / 1.27 / 1.28 but with go1.21

Reply:
ok, i will adjust that